cloud/ec2_fleet.go

```diff
@@ -390,6 +400,82 @@ func (m *ec2FleetManager) Cleanup(ctx context.Context) error {
 	return catcher.Resolve()
 }
 
+// cleanupIdleElasticIPs checks for any elastic IP addresses that are not
+// being actively used and releases them. This is a very slow operation and can
+// take several minutes.
+func (m *ec2FleetManager) cleanupIdleElasticIPs(ctx context.Context) error {
```

Comment:
Would we want to add some span tracing so we can track how long this takes maybe (since you the docs state it's slow)?

Reply:
Yeah, I was going to add Otel stuff to the cloud package in DEVPROD-16717 but wanted to keep the changes smaller.